test/correctness/simd_op_check.h

```diff
@@ -0,0 +1,301 @@
+#ifndef SIMD_OP_CHECK_H
+#define SIMD_OP_CHECK_H
+
+#include "Halide.h"
+#include "test/common/halide_test_dirs.h"
+#include "sys/time.h"
+#include <fstream>
+
+using namespace Halide;
+using namespace Halide::ConciseCasts;
+
+//namespace {
```

Comment:
(1) anonymous namespaces should ~never go in .h files
(2) commented-out code should never be left in checked-in code (unless there's a comment explaining why)

I presume this and the matching closure later were left in by accident